lib/blog-feed-loader.js

```diff
@@ -1,2 +1,6 @@
-// TODO: Replace this with a real implementation.
-module.exports = require('../test/browser/stub-blog-feed-loader');
+if (process.env.NODE_ENV !== 'production' &&
```

Comment:
Thanks @toolness for the detailed explanation above :smile: 
Sorry I got confused by the link you included in [this comment](https://github.com/mozilla/teach.webmaker.org/pull/973/files#r31464648) that's why I asked.  :smile_cat: